index.bs

```diff
@@ -3384,6 +3384,19 @@ To <dfn>locate nodes using inner text</dfn> with given |context nodes|,
 
 1. For each |context node| in |context nodes|:
 
+  1. If |context node| implements {{Document}} or {{DocumentFragment}}:
+
+    1. Let |child nodes| be an empty [=/list=].
+
+    1. For each node |child| in the <a spec=dom>children</a> of |context node|.
+
+      1. [=list/Append=] |child| to |child nodes|.
+
+    1. [=Extend=] |returned nodes| with the result of [=trying=] to [=locate nodes using inner text=]
+      with |child nodes|, |selector|, |max depth|, |match type|, |ignore case|, and |maximum returned node count|.
```

Comment:
looks like we should!

Reply:
Is max depth handling not correct currently? https://w3c.github.io/webdriver-bidi/#locate-nodes-using-inner-text it looks like it would be decremented on each iteration on context nodes not reflecting the actual depth?

Reply:
@OrKoN I wouldn't put it past me to have totally botched the `max depth` calculation. Feel free to fix as appropriate.

Reply:
Opened https://github.com/w3c/webdriver-bidi/pull/702 to address changes to innerText